### PR TITLE
improve support for Tongou TOWSMR1 Din rail switch _TZE204_kobbcyum

### PR DESCRIPTION
**Pull request by octa22**
This PR improves support for the Tongou TOWSMR1 Din rail RCBO (_TZE204_kobbcyum)

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -11552,17 +11552,166 @@ const definitions: DefinitionWithExtend[] = [
...
+        vendor: 'Tongou',
+        description: 'Single-phase multifunction RCBO (DIN Module)',
+        fromZigbee: [tuya.fz.datapoints, tuya.fz.gateway_connection_status],
+        toZigbee: [tuya.tz.datapoints],
+        configure: async (device, coordinatorEndpoint) => {
+            await tuya.configureMagicPacket(device, coordinatorEndpoint);
+            // Required to get the device to start reporting
+            await device.getEndpoint(1).command('manuSpecificTuya', 'dataQuery', {});
+        },
+        whiteLabel: [tuya.whitelabel('Tongou', 'TOWSMR1', 'Single-phase multifunction RCBO (DIN Module)', ['_TZE204_kobbcyum'])],
```
Comment (Koenkk) [inline src/devices/tuya.ts:11566]:
```suggestion
```

This can be removed

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -11552,17 +11552,166 @@ const definitions: DefinitionWithExtend[] = [
...
+            tuya.exposes.switch(),
+            e.temperature(),
+            e.current(),
+            e.power(),
+            e.voltage(),
+            e.energy(),
+            e.numeric('leakage_current', ea.STATE).withUnit('mA').withDescription('Current leakage'),
+            e
+                .enum('event', ea.STATE, [
+                    'Normal',
```
Comment (Koenkk) [inline src/devices/tuya.ts:11577]: Please make all entries lowercase

Comment (octa22) [reply src/devices/tuya.ts]: ok, made lowercase

File: src/devices/tuya.ts (hunk 1 above)
Comment (octa22) [reply src/devices/tuya.ts]: ok, removed

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**